configurations-conception-communes/pied-page-principale.md

```diff
@@ -0,0 +1,205 @@
+        <p>Sur les grands écrans, la bande principale est disposée en 3 colonnes et contient des liens vers « Toutes les
+          personnes-ressources », « Ministères et organismes » et « À propos du gouvernement ». Une petite ligne décorative sert
+          de pause avant de poursuivre avec des liens vers tous les thèmes et publics.</p>
+      </details>
+    </figure>
+  </div>
+  <div class="pattern-demo">
+    <figure class="mrgn-bttm-lg">
+      <figcaption><b>Bande principale – petit écran</b></figcaption>
+      <img src="../images/footer-main-mobile-fr.jpg" class="img-responsive"
+				alt="Schéma de la bande principale pour les petits écrans. Version texte ci-dessous&nbsp;:">
+      <details>
+        <summary class="wb-toggle" data-toggle="{&quot;print&quot;:&quot;on&quot;}">Version texte</summary>
+        <p>Sur les petits écrans, le pied de page principal est disposé en une seule colonne et contient des liens vers : « Toutes
+          les personnes-ressources », « Ministères et organismes » et « À propos du gouvernement ». Une petite ligne décorative
+          sert de pause avant de poursuivre avec des liens vers tous les thèmes et publics.</p>
+      </details>
+    </figure>
+  </div>
+</section>
+<section>
+  <h2 id="comment">Comment mettre en œuvre</h2>
+  <p>Trouvez des exemples pratiques et du code pour mettre en œuvre le pied de page.</p>
+  <h3>Référence de mise en oeuvre du thème GCWeb</h3>
+  <h4>Norme</h4>
+  <ul>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/no-footer-contextual-fr.html">Bande principale et bande sous pied de page</a></li>
+  </ul>
+  <h4>Mise en œuvre alternée pour les pages régulières</h4>
+  <ul>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/footers-fr.html">Pied de page complet (bande contextuelle, bande principale et bande sous de pied de page)</a></li>
+  </ul>
+  <h4>Mises en œuvre alternatives pour les pages transactionnelles et les pages de campagne</h4>
+  <ul>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/footers-fr.html">Pied de page complet (bande contextuelle, bande principale et bande sous de pied de page)</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/only-footer-main-fr.html">Bande principale et bande sous pied de page sans les liens facultatifs</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/no-footer-main-fr.html">Bande contextuelle et bande sous pied de page</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/only-footer-contextual-fr.html">Bande contextuelle et bande sous pied de page sans les liens facultatifs</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/only-footer-corporate-fr.html">Uniquement la bande sous pied de page</a></li>
+    <li><a href="https://wet-boew.github.io/GCWeb/sites/footers/no-footers-fr.html">Uniquement la bande sous pied de page sans liens facultatifs</a></li>
```

Comment:
Uniquement la bande sous pied de page sans les liens facultatifs

Reply:
Done.  I love making the changes this way!